lms/templates/student_account/finish_auth.html

```diff
@@ -9,32 +9,9 @@
 </%block>
 
 <%block name="headextra">
-
```

Comment:
I think this is OK since `isExternal` is getting placed in the global scope; @andy-armstrong can you confirm?